### DISCUSSION TITLE
how to import a .md file and display it as it is.

### DISCUSSION THREAD

**Discussion by ampsteric** [remarkjs/remark#704] [Q&A] [answered]
instead of writing the markdown inside the component can I just import a readymade .md file and pass it as a parameter?

**Comment by wooorm** [comment, marked as answer]
Please see the guidelines on [how to ask quality questions](https://github.com/remarkjs/.github/blob/main/support.md#asking-quality-questions)

**Reply by PaulRBerg** [reply]
Hi @wooorm, I came here having the same problem as @ampsteric. I think that his question, although short, was clear and straight to the point.

Is it possible to load Markdown content from `*.md` file and display it with `react-markdown`?

**Reply by wooorm** [reply]
This is still incredibly vague. What is “load”? From where?
For now, the answer is: you can load things from anywhere, and it is unrelated to `react-markdown`. If you can load JSON from a CMS, database, file system, server, etc, you can do the same with markdown files.

**Reply by PaulRBerg** [reply]
Load a file relative to the JavaScript React file. At any rate, I found the answer [here](https://stackoverflow.com/a/52232940/3873510). Thanks anyway!

FWIW, at least when using create-react-app, loading JSON files is a bit different from loading other files in React.
